scripts/install/opt/seagate/cortx/motr/conf/setup.yaml

```diff
@@ -45,7 +44,6 @@ motr:
     post_update:
         script: /usr/sbin/m0provision
         args:
-            - config
```

Comment:
Same here, do we need this block as ha is not going to call post_update and we are handling it in rpm update %postinstall tag ?